flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -1348,6 +1348,7 @@
                     <NODE_ID>SWITCH_OVHD_EXTLT_BEACON</NODE_ID>
                     <PART_ID>Beacon_Light_Switch</PART_ID>
                     <ANIM_NAME>SWITCH_OVHD_EXTLT_BEACON</ANIM_NAME>
+                    <TT_DESCRIPTION_ID>COCKPIT.TOOLTIPSV2.LIGHTING_SWITCH_LIGHT_BEACON_ACTION_SET</TT_DESCRIPTION_ID>
```

Comment:
Late change request (doesn't affect functionality) can you please add comments here so that we know it's a workaround and not a fix, (i.e. TODO FIXME remove workaround thanks)

Reply:
> Late change request (doesn't affect functionality) can you please add comments here so that we know it's a workaround and not a fix, (i.e. TODO FIXME remove workaround). Thanks.

Done! Good suggestion. Thanks